config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
   # Defines the root path route ("/")
-  # root "posts#index"
+  get '/users', to: 'users#index'
+
+  # root "articles#index"
+  get '/users/:id', to: 'users#show', as: 'user'
+
+  get '/user/:user_id/posts', to: 'posts#index', as: 'user_posts'
+
+  get '/user/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
- [ **OPTIONAL**] Well done adding the routes. However, I think it would be better to update your routes, as shown below. Because it's cleaner and requires less typing.

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Done, thank you 👍😃